core-services/prompt-registry/src/main/java/com/sap/ai/sdk/prompt/registry/client/OrchestrationConfigsApi.java

```diff
@@ -423,7 +421,8 @@ public OrchestrationConfigGetResponse getOrchestrationConfigByUuid(
    */
   @Nonnull
   public OrchestrationConfigPostResponse importOrchestrationConfig(
-      @Nullable final String aiResourceGroup, @Nullable final File _file)
+      @Nullable final String aiResourceGroup,
+      @Nullable final org.springframework.core.io.Resource _file)
```

Comment:
(Comment)

Usage of `org.springframework.core.io.Resource` was a compromise. In the future we may find a way to offer `File` or  `byte[]` as argument without compilation to fail. Since this is generated code we do not offer hard API contract. The argument type here may be affected by breaking changes in the future - e.g. when migrating away from RestTemplate / Spring classes.